az-core/src/main/java/azkaban/Constants.java

```diff
@@ -151,6 +151,9 @@
      *    implies save latest 3 versions saved in storage.
      **/
     public static final String AZKABAN_STORAGE_ARTIFACT_MAX_RETENTION = "azkaban.storage.artifact.max.retention";
+
+    // enable Quartz Scheduler if true.
+    public static final String ENABLE_QUARTZ= "enable.quartz";
```

Comment:
prefix it with azkaban.
Maybe
azkaban.server.schedule.enable_quartz?

Reply:
why do we need a long name? I like `azkaban.enable_quartz`?

Reply:
I am less worried about the length. We can copy and paste easily.
IMHO a more descriptive name is better. We would like to maintain a clear and consistent name hierarchy and namespace going forward.

Reply:
Updated.